src/DynamoDbQueryBuilder.php

```diff
@@ -453,40 +477,31 @@ protected function getAll($columns = [], $limit = -1, $use_iterator = true)
         if ($limit === -1 && isset($this->limit)) {
             $limit = $this->limit;
         }
-
+        $this->applyScopes();
         if ($conditionValue = $this->conditionsContainKey()) {
             if ($this->conditionsAreExactSearch()) {
                 $item = $this->find($conditionValue, $columns);
-
                 return new Collection([$item]);
             }
         }
-
```

Comment:
yeah, not sure what happened there - had some conflicts merging your changes over the weekend into the branch ... I think I copy/pasted some stuff and the newlines must have been lost in the process - sorry about that.